readme.md

```diff
@@ -354,6 +354,7 @@
 - [Search Engine Optimization](https://github.com/marcobiedermann/search-engine-optimization#readme)
 - [Competitive Programming](https://github.com/lnishan/awesome-competitive-programming#readme)
 - [Math](https://github.com/rossant/awesome-math#readme)
+	- [Topological Data Analysis](https://github.com/FatemehTarashi/awesome-TDA)
```

Comment:
> Your entry here should include a short description about the project/theme of the list. It should not describe the list itself. The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.

You could go with _"In applied mathematics, TDA is an approach to the analysis of datasets using techniques from topology."_ (which is already in the description in your readme).